ui/src/components/Accounts/AccountListSettings.tsx

```diff
@@ -37,8 +69,39 @@ const AccountListSettings = () => {
 		setNameInput(event.target.value);
 	}, []);
 
+	useEffect(() => {
+		if (injectedAccounts.length === 0) {
+			updatedInjectedAccounts();
+		}
+	}, []);
+
 	return (
 		<ModalBox title="Add New Account">
+			<p className="text-lg font-bold"> Extension Account </p>
+			<ul className="list-disc">
+				{injectedAccounts.map(({ address, name, duplicate }) => (
+					<li key={address}>
+						{address} /{name} /
+						<button
+							type="button"
+							disabled={duplicate}
+							onClick={() => addInjectedAccountToList(name, address)}
```

Comment:
@HoseinEmrani I think this now has the correct functionality, but needs some "prettification" fluff. Can you help out? I am super slow with this tailwind thing.